src/vtkF3DAssimpImporter.h

```diff
@@ -0,0 +1,127 @@
+/**
+ * @class   vtkF3DAssimpImporter
+ * @brief   Importer using Assimp library
+ * 
+ * This importer is based on Assimp 5 and supports all related file format.
+ * The list of supported file format is available here:
+ * https://github.com/assimp/assimp/blob/master/doc/Fileformats.md
+ * 
+ * The following formats have been tested and are supported by f3d: FBX, DAE, OFF, DXF
+ */
+
+#ifndef vtkF3DAssimpImporter_h
+#define vtkF3DAssimpImporter_h
+
+#include <vtkImporter.h>
+#include <vtkVersion.h>
+
+class vtkF3DAssimpImporterInternal;
+
+class vtkF3DAssimpImporter : public vtkImporter
+{
+public:
+  static vtkF3DAssimpImporter* New();
+  vtkTypeMacro(vtkF3DAssimpImporter, vtkImporter);
+  void PrintSelf(ostream& os, vtkIndent indent) override;
+
+#if VTK_VERSION_NUMBER >= VTK_VERSION_CHECK(9, 1, 0)
+  //@{
+  /**
+   * Get/Set the file name.
+   */
+  vtkSetStdStringFromCharMacro(FileName);
+  vtkGetCharFromStdStringMacro(FileName);
+  //@}
+#else
+  /**
+   * Get the filename
+   */
+  const char* GetFileName()
+  {
+    return this->FileName.c_str();
+  }
+
+  /**
+   * Set the filename
+   */
+  void SetFileName(const char* fileName)
+  {
+    this->FileName = fileName;
+    this->Modified();
+  }
+#endif
+
+  /**
+   * Update actors at the given timestep.
+   */
+  void UpdateTimeStep(double timestep) override;
+
+  /**
+   * Get the number of available animations.
+   */
+  vtkIdType GetNumberOfAnimations() override;
+
+  /**
+   * Return the name of the animation.
+   */
+  std::string GetAnimationName(vtkIdType animationIndex) override;
+
+  //@{
+  /**
+   * Enable/Disable/Get the status of specific animations
+   */
+  void EnableAnimation(vtkIdType animationIndex) override;
+  void DisableAnimation(vtkIdType animationIndex) override;
+  bool IsAnimationEnabled(vtkIdType animationIndex) override;
+  //@}
+
+  /**
+   * Return importer description.
+   */
+  std::string GetOutputsDescription() override;
+
+#if VTK_VERSION_NUMBER >= VTK_VERSION_CHECK(9, 1, 0)
```

Comment:
more precise version check ?

Reply:
Do we really want to support versions between 9.0 and 9.1?

Reply:
we have many checks between 9.0 and 9.1, so unless there is something I'm missing, it may be better to check the right version as it could help us understand which commit in VTK is related to this.
If you dont know and it requires digging in VTK history, I'm fine to keep as is.